sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -388,4 +388,19 @@ class QueryExecutionErrorsSuite extends QueryTest
             |""".stripMargin)
     }
   }
+
+  test("FAILED_EXECUTE_UDF: execute user defined function") {
+    val e1 = intercept[SparkException] {
+      val names = Seq("Jacek", "Agata", "Sweet").toDF("name")
+      val hello = udf { _: String => {
+        throw new SparkException("throw spark exception for udf test")
+      }}
+      names.select(hello($"name")).collect()
+    }
+    assert(e1.getCause.isInstanceOf[SparkException])
+
+    val e2 = e1.getCause.asInstanceOf[SparkException]
+    assert(e2.getErrorClass === "FAILED_EXECUTE_UDF")
+    assert(e2.getMessage.startsWith("Failed to execute user defined function "))
```

Comment:
Please, compare to the entire error message. A purpose is to document errors in tests.

Reply:
ok, i will use regex match it.